kong/conf_loader/parse.lua

```diff
@@ -432,6 +432,20 @@ local function check_and_parse(conf, opts)
         conf.ssl_dhparam = suite.dhparams
         conf.nginx_http_ssl_dhparam = suite.dhparams
         conf.nginx_stream_ssl_dhparam = suite.dhparams
+      else
+        for _, key in ipairs({
+          "nginx_http_ssl_conf_command",
+          "nginx_http_proxy_ssl_conf_command",
+          "nginx_http_lua_ssl_conf_command",
+          "nginx_stream_ssl_conf_command",
+          "nginx_stream_proxy_ssl_conf_command",
+          "nginx_stream_lua_ssl_conf_command"}) do
+
+          local _, _, seclevel = string.find(conf[key] or "", "@SECLEVEL=(%d+)")
+          if conf[key] and seclevel ~= "0" then
+            ngx.log(ngx.WARN, key .. ": Default @SECLEVEL=0 overridden, TLSv1.1 unavailable")
```

Comment:
Do not use concat `..`,  in `ngx.log` we could log multiple values, like `ngx.log(warn, a, b, c)`

Reply:
> should be find(str, x, 1, true)

@chronolaw, I think that one uses a pattern. So in this case it should not.

Reply:
Oh, sorry, it is my mistake, thank you.